src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <Compile Include="Perf.Span.BinaryReadAndWrite.cs" />
     <Compile Include="Perf.Span.BinarySearch.cs" />
     <Compile Include="Perf.Span.Clear.cs" />
+    <None Include="Perf.Span.Contains.cs" />
```

Comment:
Not sure where to put these tests such that they don't get broken by cross-compilation (eg NetFx)

Reply:
Hmm, this one is a first I think. Add a conditional ItemGroup which is try only when the configuration is netcoreapp only and add it to that?

Reply:
Removed this class per separate feedback

Reply:
I must have misunderstood you, it sounded like the benchmarks were not as useful without a prior baseline:

> What are we trying to measure with this test? Do we a have a baseline to compare the result against?

Reply:
I will add it back

Reply:
> I must have misunderstood you, it sounded like the benchmarks were not as useful without a prior baseline

I was talking about that specific benchmark (only): `SpanContainsSpanComparison`